backend/src/routes/api/configmaps/index.ts

```diff
@@ -0,0 +1,54 @@
+import { KubeFastifyInstance } from '../../../types';
+import { FastifyRequest } from 'fastify';
+import { V1ConfigMap } from '@kubernetes/client-node';
+
+module.exports = async (fastify: KubeFastifyInstance) => {
+  fastify.get('/:name', async (request: FastifyRequest) => {
+    const params = request.params as {
+      name: string;
+    };
+    try {
+      const cmResponse = await fastify.kube.coreV1Api.readNamespacedConfigMap(
+        params.name,
+        fastify.kube.namespace,
+      );
+
+      return cmResponse.body.data;
```

Comment:
Change here to `cmResponse.body` instead, because data may not exist if this is an empty config map, and fastify will give you an error when you return an `undefined`.